core/src/main/java/me/mykindos/betterpvp/core/combat/damagelog/DamageLogListener.java

```diff
@@ -4,18 +4,25 @@
 import com.google.inject.Singleton;
 import me.mykindos.betterpvp.core.framework.updater.UpdateEvent;
 import me.mykindos.betterpvp.core.listener.BPvPListener;
+import me.mykindos.betterpvp.core.utilities.UtilMessage;
+import net.kyori.adventure.text.Component;
+import net.kyori.adventure.text.event.ClickCallback;
+import net.kyori.adventure.text.event.ClickEvent;
+import net.kyori.adventure.text.format.NamedTextColor;
+import org.bukkit.entity.Player;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.EventPriority;
 import org.bukkit.event.Listener;
+import org.bukkit.event.entity.PlayerDeathEvent;
+
+import java.util.concurrent.ConcurrentLinkedDeque;
 
 @Singleton
 @BPvPListener
 public class DamageLogListener implements Listener {
 
-    private final DamageLogManager damageLogManager;
-
     @Inject
-    public DamageLogListener(DamageLogManager damageLogManager) {
-        this.damageLogManager = damageLogManager;
-    }
+    private DamageLogManager damageLogManager;
```

Comment:
Why?

Reply:
Did it for cleanliness since I saw other classes like commands avoid creating a constructor if Guice can directly inject into their field member. I can remove it

Reply:
Burning the steak, it's only commands that do it I believe